src/odemis/acq/align/test/goffset_test.py

```
@@ -66,37 +170,12 @@ def setUp(self):
-        image = np.tile(line, (50, 1))
-
-        peak = find_peak_position(image)
-        self.assertAlmostEqual(peak, true_center, places=1)
-
     @timeout(100)
     def test_estimate_goffset_scale(self):
         """
         Test that goffset scale is non-zero and finite.
         """
-        scale = estimate_goffset_scale(self.spgr, self.detector)
+        scale, p0, p1 = estimate_goffset_scale(self.spgr, self.detector)
```

Ruff (0.15.18)

[warning] Unpacked variable `p0` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)